docs/mocksupport.rst

```diff
@@ -600,6 +600,11 @@ There are two ways to build a mock repository:
   * Build from MOF definitions of the objects which are compiled into the
     repository. See :meth:`~pywbem_mock.FakedWBEMConnection.compile_mof_string`
     and See :meth:`~pywbem_mock.FakedWBEMConnection.compile_mof_file`.
+    If instances are to be compiled "instance of ..." they must be compiled
+    with the instance alias (ex:"instance of CIM_xxx as $xxx {") which adds
+    the CIMInstanceName with keybindings from the key properties to the
+    instance path. If compiled instances have duplicate paths, each new
+    instance will modify the already created instance.
```

Comment:
change "already created" to "already existing".

Reply:
Agreed. DONE